### PR TITLE
OPSEXP-1294 Update README for new tag naming convention

### PR DESCRIPTION
**Pull request by gionn**
OPSEXP-1294

Preview: https://github.com/Alfresco/alfresco-docker-base-tomcat/blob/OPSEXP-1294-readme-rewrite/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,52 @@
...
 
-The builds are identical to those stored in the private repo on Quay, which also supports build-pinning versions.
+Used as parent image and with the default configuration, ensure the following
+volumes are all specified.
 
 ```bash
-docker pull quay.io/alfresco/alfresco-base-tomcat:$TOMCAT_MAJOR_MINOR_VERSION
-docker pull quay.io/alfresco/alfresco-base-tomcat:$TOMCAT_VERSION-java-$JAVA_MAJOR-$DISTRIB_NAME-$DISTRIB_MAJOR
-docker pull quay.io/alfresco/alfresco-base-tomcat:$TOMCAT_VERSION-java-$JAVA_MAJOR-$DISTRIB_NAME-$DISTRIB_MAJOR-$SHORT_SHA256
+VOLUME [ "/usr/local/tomcat/logs", "/usr/local/tomcat/work", "/usr/local/tomcat/conf/Catalina", "/usr/local/tomcat/temp" ]
```
Comment (alxgomz) [inline README.md:49]: I'm not sure I understand the purpose of that. logs management should be left to the user for him to choose his logging config.
Why would `work` and `temp` need to be volume mounted?
Regarding `conf/Catalina` I can see the attemped goal but there very few wsettings one would ever change here and - for now at least - we recommend people should create a new image from ours if they needs to tweak that.

File: README.md [hunk 2]
```diff
@@ -85,60 +63,44 @@ With the supplied tomcat configuration, the following should all be mounted on v
...
-instruction,
-[best practices with VOLUMEs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#volume),
-[best practices with persistence](https://docs.docker.com/develop/dev-best-practices/#where-and-how-to-persist-application-data),
-and
-[tmpfs](https://docs.docker.com/storage/tmpfs/) mounts.
-
-### Examples of usage as a parent image
-
-Example from a Dockerfile using a public parent image in Docker Hub.
+The Tomcat in this image is running with Security Manager switched on. This may impact performance. The Security Manager can be disabled by overriding the startup command to:
```
Comment (alxgomz) [inline README.md:69]: I don't think we should document how to reduce security.

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gionn) [reply README.md:49]: this and the next section is copied as is and it's here from [4 years ago](https://github.com/Alfresco/alfresco-docker-base-tomcat/blame/master/README.md#L74) - I've just moved them around the README

I don't know if these suggestions really provide some value worth documenting here, however at this time there is no volume defined in Dockerfile and I don't really know if the user would expect having such folders in volumes or not

File: README.md [hunk 3]
```diff
@@ -1,74 +1,52 @@
...
+This image depends on the [alfresco-docker-base-java](https://github.com/Alfresco/alfresco-docker-base-java).
 
-## Versioning
+## Quickstart
 
-Images are available for latest Tomcat 8.5.76 and 9.0.59 (last two are Java 11 only).
+Choose between one of the available flavours built from this repository:
 
-## How to Build
+Tomcat version | Java version | Java flavour | OS       | Image tag     | Size
```
Comment (mteodori) [inline README.md:16]: align the last column so it's readable also in source form

File: README.md [hunk 4]
```diff
@@ -1,74 +1,52 @@
...
+---------------|--------------|--------------|----------|---------------|-------------------------
+9              | 11           | jre          | Centos 7 | tomcat9-jre11-centos7 | ![tomcat9-jre11-centos7 size][1]
+9              | 11           | jdk          | Centos 7 | tomcat9-jdk11-centos7 | ![tomcat9-jdk11-centos7 size][2]
+8              | 11           | jre          | Centos 7 | tomcat8-jre11-centos7 | ![tomcat8-jre11-centos7 size][3]
+8              | 11           | jdk          | Centos 7 | tomcat8-jdk11-centos7 | ![tomcat8-jdk11-centos7 size][4]
+9              | 11           | jre          | Ubi 8    | tomcat9-jre11-ubi8    | ![tomcat9-jre11-ubi8 size][5]
```
Comment (mteodori) [inline README.md:22]: UBI capitalised

File: README.md [hunk 5]
```diff
@@ -1,74 +1,52 @@
...
+[4]: https://img.shields.io/docker/image-size/alfresco/alfresco-base-tomcat/tomcat8-jdk11-centos7
+[5]: https://img.shields.io/docker/image-size/alfresco/alfresco-base-tomcat/tomcat9-jre11-ubi8
+[6]: https://img.shields.io/docker/image-size/alfresco/alfresco-base-tomcat/tomcat8-jre11-ubi8
 
-* (privately) on [Quay](https://quay.io/repository/alfresco/alfresco-base-java)
-* (publicly) on [Docker Hub](https://hub.docker.com/r/alfresco/alfresco-base-java)
+* [Docker Hub](https://hub.docker.com/r/alfresco/alfresco-base-tomcat) image name: `alfresco/alfresco-base-tomcat`
+* [Quay](https://quay.io/repository/alfresco/alfresco-base-tomcat) image name: `quay.io/alfresco/alfresco-base-tomcat`
 
-To build this image, run the following script:
```
Comment (mteodori) [inline README.md:22]: I would local build info

File: README.md (hunk 1 above)
Comment (mteodori) [reply README.md:49]: can be removed

File: README.md (hunk 2 above)
Comment (mteodori) [reply README.md:69]: security manager is deprecated and can be removed, maybe in another PR https://openjdk.java.net/jeps/411

File: README.md (hunk 1 above)
Comment (mteodori) [reply README.md:49]: @nicdoye do you know which was the rationale behind this requirement to configure volumes? it's not used in compose or charts so it hardly looks like a requirement but more of a convenience to see files on the local machine when developing

File: README.md (hunk 5 above)
Comment (gionn) [reply README.md:22]: I moved them down in the page

File: README.md (hunk 1 above)
Comment (gionn) [reply README.md]: I've simplified the paragraph stating that volumes are not really mandatory

**Review by alxgomz — APPROVED** [review]
LGTM

**Review by mteodori — APPROVED** [review]

**Pull request merged**